interface/forms/soap/templates/general_new.html

```diff
@@ -1,52 +1,58 @@
     <div class="container">
         <div class="row">
             <div class="col-xs-12">
                 <div class="page-header">
-                    <h1>{xl t='SOAP'|escape:'html'}</h1>
+                    <h2>{xl t='SOAP'|escape:'html'}</h2>
                 </div>
             </div>
         </div>
         <div class="row">
             <div class="col-xs-12">
-                <form name="soap" class="form-horizontal" method="post" action="{$FORM_ACTION}/interface/forms/soap/save.php" onsubmit="return top.restoreSession()">
-                    <div class="form-group">
-                        <label for="subjective" class="control-label col-sm-2">{xl t='Subjective'|escape:'html'}</label>
-                        <div class="col-sm-10">
-                            <textarea name="subjective" id="subjective" class="form-control" cols="60" rows="6">{$data->get_subjective()|escape:'html'}</textarea>
-                        </div>
-                    </div>
-                    <div class="form-group">
-                        <label for="objective" class="control-label col-sm-2">{xl t='Objective'|escape:'html'}</label>
-                        <div class="col-sm-10">
-                            <textarea name="objective" id="objective" class="form-control" cols="60" rows="6">{$data->get_objective()|escape:'html'}</textarea>
-                        </div>
-                    </div>
-                    <div class="form-group">
-                        <label for="assessment" class="control-label col-sm-2">{xl t='Assessment'|escape:'html'}</label>
-                        <div class="col-sm-10">
-                            <textarea name="assessment" id="assessment" class="form-control" cols="60" rows="6">{$data->get_assessment()|escape:'html'}</textarea>
-                        </div>
-                    </div>
-                    <div class="form-group">
-                        <label for="plan" class="control-label col-sm-2">{xl t='Plan'|escape:'html'}</label>
-                        <div class="col-sm-10">
-                            <textarea name="plan" id="plan" class="form-control" cols="60" rows="6">{$data->get_plan()|escape:'html'}</textarea>
-                        </div>
-                    </div>
-                    <div class="form-group">
-                        <div class="col-sm-offset-2 col-sm-10">
-                            <div class="btn-group" role="group">
+                <form name="soap" method="post" action="{$FORM_ACTION}/interface/forms/soap/save.php" onsubmit="return top.restoreSession()">
+                    <fieldset>
+                            <legend>{xl t='Subjective'|escape:'html'}</legend>
+                                <div class="form-group" >
+                                    <div class="col-sm-10 col-sm-offset-1">
+                                        <textarea name="subjective" class="form-control" cols="40" rows="6">{$data->get_subjective()|escape:'html'}</textarea>
```

Comment:
Albeit bootstrap doesn't seem to care, to be consistent with the other elements on this page, the cols above should be 60